src/main/scala/ru/wow42/tasks/futures/FutureTask.scala

```diff
@@ -1,13 +1,15 @@
+package ru.wow42.tasks.futures
+
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
-case class Test[A]() {
+
+case class FutureTask[A]() {
```

Comment:
Why you put the parameter `A` to the class?
It'll be better if we move it to the method.
Currently, every time when we need to use `travers` for list with different futures (Future[Int] and Future[String] for example) we must create a new instance of the` FutureTask`. It's not effective.